test/integration/multicluster/multicluster-traffic/mc_traffic_test.go

```diff
@@ -276,6 +277,11 @@ func TestMulticlusterStatefulSetTargetTraffic(t *testing.T) {
 			}
 		})
 
+		_, err := TestHelper.KubectlWithContext("", contexts[testutil.TargetContextKey], "--namespace="+ns, "label", "svc", "nginx-statefulset-svc", k8s.DefaultExportedServiceSelector+"=true")
```

Comment:
should we add a test for unexporting too?

Reply:
Yep, we can add that as part of the fix for #8475